index.html

```diff
@@ -47,14 +48,14 @@ <h2>Security</h2>
 <h2>Privacy Protections Tests</h2>
 
 <ul>
-	<li><a href='https://privacy-test-pages.glitch.me/privacy-protections/request-blocking/'>Request blocking</a></li>
-	<li><a href='https://privacy-test-pages.glitch.me/privacy-protections/fingerprinting/'>Fingerprinting</a></li>
+	<li><a href='./privacy-protections/request-blocking/'>Request blocking</a></li>
```

Comment:
to avoid issues with local development. This was full urls for two reasons: so that people don't try to use those pages on gh-pages deployment because they may not work and because HTTPS test page has to be loaded over HTTP to work (I fixed it with JS, see next files)